src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/PublicationRepositoryTests.cs

```diff
@@ -12,21 +12,102 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+                            Releases = new List<Release>
+                            {
+                                new Release
+                                {
+                                    Id = Guid.NewGuid(),
+                                    ReleaseName = "2000",
+                                    TimePeriodCoverage = TimeIdentifier.Week1,
+                                    Published = DateTime.UtcNow
+                                },
+                                new Release
+                                {
+                                    Id = Guid.NewGuid(),
+                                    ReleaseName = "2000",
+                                    TimePeriodCoverage = TimeIdentifier.Week11,
+                                    Published = DateTime.UtcNow
+                                },
+                                new Release
+                                {
+                                    Id = Guid.NewGuid(),
+                                    ReleaseName = "2000",
+                                    TimePeriodCoverage = TimeIdentifier.Week3,
+                                    Published = DateTime.UtcNow
+                                },
+                                new Release
+                                {
+                                    Id = Guid.NewGuid(),
+                                    ReleaseName = "2000",
+                                    TimePeriodCoverage = TimeIdentifier.Week2,
+                                    Published = DateTime.UtcNow
+                                },
+                                new Release
+                                {
+                                    Id = Guid.NewGuid(),
+                                    ReleaseName = "2001",
+                                    TimePeriodCoverage = TimeIdentifier.Week1,
+                                    Published = DateTime.UtcNow
+                                },
+                                new Release
+                                {
+                                    Id = Guid.NewGuid(),
+                                    ReleaseName = "1999",
+                                    TimePeriodCoverage = TimeIdentifier.Week1,
+                                    Published = DateTime.UtcNow
+                                }
+                            }
+                        }
+                    }
+                });
+                context.SaveChanges();
+            }
+
+            using (var context = InMemoryApplicationDbContext("ReleasesCorrectlyOrdered"))
+            {
+                var publicationService = new PublicationRepository(context, AdminMapper());
+                var publications = await publicationService.GetAllPublicationsForTopicAsync(topicId);
+                var releases = publications.Single().Releases;
+                Assert.True(releases[0].Title.Equals("Week 1 2001"));
```

Comment:
Better to use `Assert.Equals("Week 1 2001", releases[0].Title)`.

`Assert.True` is only really good for asserting on boolean values. The main problem is that when it fails, the error message will only contain limited information, rather than exactly what was wrong with the value (which `Assert.Equals` does).

Same applies for the rest of these.